### ISSUE TITLE
Is the screenshoot putty?

### ISSUE THREAD

**Issue by Roki100** [NeonWizard/spookdev-motd#1]
Hello, is the screenshoot of how it looks a putty window(kinda looks like it)? if so, can you link the colors configuration of it? also, is the font the default(also doesnt look like it)?
I wanna make the same color palette as the one on the screenshot as the red&white seems to look clean, and i want to make something similar

**Comment by NeonWizard** [comment]
It should look the same in a PuTTY terminal, yeah. The font is just monospace I believe, it's whatever the Hyper.JS terminal uses. It should appear the same in all SSH terminals though. The colors are set by using bash color formatting, I have them defined in [this file](https://github.com/NeonWizard/spookdev-motd/blob/master/motd/colors.txt).

You can find more info on bash text coloring here: https://misc.flogisoft.com/bash/tip_colors_and_formatting

**Comment by Roki100** [comment]
I am not sure what font hyper uses, but it kinda looks like consolas, is it consolas?
Also no, the screenshot appearing to be putty surely had some configuration change in putty colorset aswell as the color of the cursor is green unless changed, on the screenshot its purple, so the question is, did you change the colorset in putty itself(more probable) or bash itself
you don't use this colorset&font combo anymore? 👀

**Comment by NeonWizard** [comment]
I don't use PuTTY. I use the Hyper.JS terminal with a bash shell through WSL. The purple cursor is default through Hyper.JS. Otherwise there is no color changes except through Bash itself with the color formatting done as described in the link I sent.

**Comment by Roki100** [comment]
Oh okay, this is interesting as it actually looks like putty 👀
Thank you, btw, what hyper theme is that? Or it is the default theme? Not sure what the default looks like as last time i used Hyper was long ago, and mine is full green

**Comment by NeonWizard** [comment]
I'm pretty sure it's just the default theme

**Comment by Roki100** [comment]
Alright thank you so much, so basically the color palette was changed via .bashrc file using colors provided in colors.txt? or i got it wrong
gotta try to reproduce yours as its clean 👀😎
also ty for making&publishing this motd as it is excatly what i was looking for

**Comment by NeonWizard** [comment]
The color is changed when the text is outputted through the MOTD code. Again, I highly suggest actually reading the link I sent about color formatting. There is an example at the top of the page on how to color outputted text in the bash shell.

**Comment by Roki100** [comment]
no i actually meant the overall color palette of your terminal e.g. red username(or thats hostname, dunno), i want to replicate it

**Comment by NeonWizard** [comment]
Yes, the red username, blue directory, white and red MOTD text, etc are all colored via bash formatting. Please read the link I sent for more information.
